lib/src/executables/dart2_upgrade.dart

```diff
@@ -61,6 +69,8 @@ void main(List<String> args) {
     ),
     args: args,
     defaultYes: true,
+    additionalHelpOutput: argParser.usage,
+    changesRequiredOutput: _changesRequiredOutput,
   );
 
   if (exitCode > 0) {
```

Comment:
Instead of wrapping the second phase below in an if block, can we add to this existing if block since it already returns early? That should have the same effect and fix that double-help-output bug.

```suggestion
  if (exitCode > 0 || args.contains(_helpFlag) || args.contains('-h')) {
```